src/main/java/io/github/thebusybiscuit/slimefun4/implementation/setup/SlimefunItemSetup.java

```diff
@@ -1016,6 +1017,10 @@ public static void setup(@Nonnull SlimefunPlugin plugin) {
         new PotionEffect[] {new PotionEffect(PotionEffectType.NIGHT_VISION, 600, 20)})
         .register(plugin);
 
+        new ElytraCap(categories.technicalGadgets, SlimefunItems.ELYTRA_CAP, RecipeType.ARMOR_FORGE,
```

Comment:
Why is this item in Technical Gadgets category? Is it random?

Reply:
On discord it was suggested to put in technical gadgets